packages/mdc-select/label/_mixins.scss

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/theme/mixins";
+
+@mixin mdc-select-floating-label-color($color, $opacity: 1) {
```

Comment:
Should this be private ?

Reply:
In the current implementation in #2237 it is public. It needs to be public for other modules to change the color. Why do you think it needs to be private?

Reply:
I thought public color mixins were supposed to explicitly prevent styling the disabled state 
`:not(.mdc-text-field--disabled)`. 

https://github.com/material-components/material-components-web/blob/3b0977d60fe6c94528bb348bff5b17edf460ff1f/packages/mdc-textfield/label/_mixins.scss#L22

That doesn't really make sense for the separate package though, so maybe it doesn't matter.

Reply:
I think it doesn't make sense to have that in the public mixin, but I guess the question then is, should we allow the end developer to style the color in the disabled state?

Also in the current state being in the label module, we shouldn't add in parent classes like .mdc-text-field--disabled or .mdc-select--disabled

Reply:
I think we're explicitly preventing them from styling the disabled state. It doesn't make sense to include the parent class, we added proxy mixins from the parent to the line-ripple module to prevent customizing the disabled state.